general/naming.md

```diff
@@ -0,0 +1,112 @@
+# Naming guidelines
+
+> There are only two hard things in Computer Science: cache invalidation and naming things.
+
+([Source](https://skeptics.stackexchange.com/a/39178))
+
+## General naming advice
+
+### Avoid abbreviations
+
+Using abbreviations can make code harder to understand, especially on an international team where English may not be everybody's first language.
+
+Thanks to tools like [Prettier](https://prettier.io/) that can wrap code neatly and readably, we don't have to worry so much about minimizing line width. We can instead prioritize encoding information clearly, accurately, and unambiguously.
+
+#### Examples of _bad_ abbreviations:
+
+| Abbreviations      | Preferred alternatives                      |
+| ------------------ | ------------------------------------------- |
+| :x: `arr`          | :white_check_mark: `array`                  |
+| :x: `btn`          | :white_check_mark: `button`                 |
+| :x: `cb`           | :white_check_mark: `callback`               |
+| :x: `desc`         | :white_check_mark: `description`            |
+| :x: `e`/`err`      | :white_check_mark: `error`                  |
+| :x: `el`           | :white_check_mark: `element`                |
+| :x: `evt`          | :white_check_mark: `event`                  |
+| :x: `fm`           | :white_check_mark: `form`                   |
+| :x: `fmt`          | :white_check_mark: `format`                 |
+| :x: `k`/`v`        | :white_check_mark: `key`/`value`            |
+| :x: `idx`          | :white_check_mark: `index`                  |
+| :x: `img`          | :white_check_mark: `img`                    |
```

Comment:
I think you probably meant  `:white_check_mark: image`

Reply:
Good `catch`! Thanks.

![our-first-catch-of-the-day](https://user-images.githubusercontent.com/7074/92461521-af4cfc00-f1c9-11ea-9d09-703858c8e6b9.jpg)

Reply:
Fixed in: https://github.com/liferay/liferay-frontend-guidelines/pull/195/commits/db9dc2125f74f1d74caf44d6144483982ff006ad

Reply:
:tada:

Reply:
It's nice to have both **good** and **bad** examples - never would have figured out that `prj` stands for `project`